config/application.rb

```diff
@@ -9,19 +9,19 @@
 module Sulbib
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 6.0
-    config.autoloader = :zeitwerk
-    config.eager_load_paths << Rails.root.join('lib')
+    config.load_defaults 7.2
 
-    # Settings in config/environments/* take precedence over those specified here.
-    # Application configuration can go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded after loading
-    # the framework and any gems in your application.
+    # Please, add to the `ignore` list any other `lib` subdirectories that do
+    # not contain `.rb` files, or that should not be reloaded or eager loaded.
+    # Common ones are `templates`, `generators`, or `middleware`, for example.
+    config.autoload_lib(ignore: %w[assets tasks])
 
-    config.active_job.queue_adapter = :async
-
-    # https://discuss.rubyonrails.org/t/cve-2022-32224-possible-rce-escalation-bug-with-serialized-columns-in-active-record/81017
-    # Used in publications.pub_hash
-    config.active_record.yaml_column_permitted_classes = [Symbol, Hashie::Mash, Hashie::Array]
```

Comment:
Sure you don't need this anymore?

Reply:
thanks ... no, i'm not sure -- so i will put it back just in case